docs/index.adoc

```diff
@@ -3003,9 +3003,10 @@ Methods annotated with `@Option` and `@Parameters` can do simple input validatio
 
 The following example validates that the value specified for the `--prime` option is a prime number:
 
-[source,java]
+.Java
+[source,java,role="primary"]
 ----
-class SingleOptionValidationExample {
+class SingleOptionValidationExample implements Runnable {
```

Comment:
I was thinking to leave out the `implements Runnable`.  The main point of this example is that apps can use an `@Option`-annotated method to do validation. 

I think it is okay to for some examples to be fragments (as opposed to a fully executable program). Sometimes that helps to keep the example focused and communicates the point better.